Sources/docc/DocCDocumentation.docc/adding-structure-to-your-documentation-pages.md

```diff
@@ -206,6 +210,11 @@ documentation hierarchy.
 
 After you arrange nested symbols in an extension file, use DocC to compile your changes and review them in your browser.
 
+> Note: 
+> If you organize a member symbol into a topic group outside of its canonical container symbol, 
+> DocC will still include the member symbol in the default topic group.
+> This ensures that the reader can always find the member symbol somewhere in the sub-hierarchy of its canonical container symbol. 
```

Comment:
An example here might be helpful, as I'm not sure how recognisable the term "canonical container symbol" will be for those reading this documentation.

Reply:
I rephrased this in https://github.com/swiftlang/swift-docc/pull/1075/commits/f78895066bdbde0b74cad6c5bf1087e60eb302bf to "the type that defines the member". That sentence gets a bit confusing with symbols that have different locations in different language representations but I think overall fewer people will be confused by it.